### PR TITLE
Python 3.9 compatability

### PR DESCRIPTION
**Pull request by sam-lawrence**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -15,6 +15,9 @@ lxml = "^4.9.2"
 [tool.poetry.group.dev.dependencies]
 mypy = "^1.1.1"
 flake8 = "^6.0.0"
+types-beautifulsoup4 = "^4.12.0.3"
+pandas-stubs = "^2.0.0.230412"
```
Comment (risktoparkrageram) [inline pyproject.toml:33]: should we bump pandas to ^2 as well?

**Review by risktoparkrageram — APPROVED** [review]
LGTM

File: pyproject.toml (hunk 1 above)
Comment (sam-lawrence) [reply pyproject.toml:33]: If it works with ^1 I'm tempted to leave it.  It should stlil give the latest on any new install

**Comment by sam-lawrence** [comment]
Thanks @risktoparkrageram @janrito !

**Pull request merged**